backend/items/services.py

```
@@ -314,3 +321,585 @@ def check_and_send_only_due_emails(user):
+        ).order_by('full_path')
+    
+    @staticmethod
+    def get_items_at_location(location, include_descendants=False):
+        """
+        Get items at a specific location
+        If include_descendants=True, include items from all child locations
+        """
+        if include_descendants:
+            # Get items at this location and all descendants
+            descendant_locations = location.get_descendants()
+            location_ids = [location.id] + list(descendant_locations.values_list('id', flat=True))
```

Ruff (0.15.2)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)